pom.xml

```diff
@@ -1706,6 +1706,43 @@
         <jline.groupid>jline</jline.groupid>
       </properties>
     </profile>
+    <profile>
+        <id>log4j2</id>
+        <activation>
+            <property><name>log4j2</name></property>
+        </activation>
+        <properties>
+            <log4j2.version>2.1</log4j2.version>
+        </properties>
+        <dependencies>
```

Comment:
Hm. I wonder what would happen if Spark always used log4j 2.x, and then included the 1.2 -> 2 bridge code just in case an app relies on Spark providing 1.2?